test/Worker.Extensions.Tests/ServiceBus/ServiceBusSessionMessageActionsTests.cs

```diff
@@ -49,6 +50,55 @@ public async Task CanRenewSessionLock()
             await messageActions.RenewSessionLockAsync();
         }
 
+        [Fact]
+        public async Task CanSetAndGetSessionStateWithNull()
+        {
+            var sessionId = "test";
+            var initialTestString = "TestInitialState";
+            ByteString currentSessionState = ByteString.CopyFromUtf8(initialTestString);
+
+            var mockClient = new Mock<Settlement.SettlementClient>();
+            
+            mockClient
+                .Setup(x => x.GetSessionStateAsync(
+                    It.Is<GetSessionStateRequest>(r => r.SessionId == sessionId),
+                    It.IsAny<Metadata>(),
+                    It.IsAny<DateTime?>(),
+                    It.IsAny<CancellationToken>())
+                )
+                .Returns(() => CreateAsyncUnaryCall(new GetSessionStateResponse
+                {
+                    SessionState = currentSessionState
+                }));
+
+            mockClient
+                .Setup(x => x.SetSessionStateAsync(
+                    It.Is<SetSessionStateRequest>(r => r.SessionId == sessionId),
+                    It.IsAny<Metadata>(),
+                    It.IsAny<DateTime?>(),
+                    It.IsAny<CancellationToken>())
+                )
+                .Returns<SetSessionStateRequest, Metadata, DateTime?, CancellationToken>((request, _, _, _) =>
+                {
+                    currentSessionState = request.SessionState;
+                    return CreateAsyncUnaryCall(new Empty());
+                });
+
+            var message = ServiceBusModelFactory.ServiceBusReceivedMessage(lockTokenGuid: Guid.NewGuid(), sessionId: sessionId);
+            var messageActions = new ServiceBusSessionMessageActions(settlement: mockClient.Object,sessionId: message.SessionId,sessionLockedUntil: message.LockedUntil);
+
+            var initialState = await messageActions.GetSessionStateAsync();
+            Assert.Equal(initialTestString, initialState.ToString());
+
+            await messageActions.SetSessionStateAsync(null);
+            var afterSetNullState = await messageActions.GetSessionStateAsync();
+            Assert.Null(afterSetNullState);
+        }
+        private static AsyncUnaryCall<T> CreateAsyncUnaryCall<T>(T response) where T : class
```

Comment:
nit, newline after `}`
```suggestion

        private static AsyncUnaryCall<T> CreateAsyncUnaryCall<T>(T response) where T : class
```